pkgs/tools/games/ajour/default.nix

```diff
@@ -0,0 +1,81 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, autoPatchelfHook
+, cmake
+, makeWrapper
+, pkg-config
+, python3
+, expat
+, freetype
+, gnome3
+, plasma5Packages
```

Comment:
Please don't input plasma5Packages. It should be inherit from all-packages.

Reply:
Can you reference some docs? Why is it fine for gnome3?

Reply:
I saw that on a recent comment from ttuegel but I probably won't find it.